### ISSUE TITLE
Hide watched videos / mark videos as watched

### ISSUE THREAD

**Issue by Creat1veM1nd** [lamarios/clipious#299]
Hello,

on the instance I use, there is an option to hide watched videos and also to mark them as watched by clicking an icon in the overview. That helps a lot to organize subscriptions. Is there a chance to implement this ? In sync with the connected instance would be great !

Thanks in advance !

**Comment by Malakai-DB** [comment]
I am not sure I understand the request.
Is it about the fact that when you watch a video from your subscriptions, it is not marked as watched in invidious?
If I am understanding correctly, then whats up with that?
I would have expected for clipious to be synchronized with invidious regarding this.
Why isn't it the case?
The way things are working now, after watching videos from my subscriptions on clipious from my phone, when I access the same instance of invidious on my desktop I have to cross check which videos were watched and mark them as watched there also. Not too pleasant.
Is this feature request in sight to be implemented?

**Comment by noticons** [comment]
My understanding of this option would be to have the option in the menu where you can press "Add to Playlist" "Add to queue" "Play next" "Download" and "Share" that there would be another option to just "Mark video as watched" which would then mark it as watched in the feed and on the backend, as well.

**Comment by noticons** [comment]
@Malakai-DB yes, that's what happens on my invidious instance. I run my own instance, though, with a login and I use clipious with a token to keep logged in to my instance. You may be using a public instance? 
So, in this sense, my understanding of this request is that I can tell my instance to mark a video as watched, even if I haven't watched it on that instance, or whatnot.

**Comment by Malakai-DB** [comment]
@noticons 
> yes, that's what happens on my invidious instance. I run my own instance, though, with a login and I use clipious with a token to keep logged in to my instance.
Really? I run my own instance too, also with login and clipious has a token also, but I am not experiencing this. Is there anything to configure for this (I didn't see anything while trying to understand the behavior)?

Just to be sure : I use mainly 2 views on my instance, the one with my subscriptions (/feed/subscriptions) and the one with the videos I already watched (/feed/history). When I watch a video from my subscriptions on invidious, it shows afterwards in the history view. If I watch a video (also from my subscriptions) on clipious, I see a mark meaning that the video was watched, but the video stays in my subscriptions view. If I open invidious, the video is still in my subscriptions and not in my history view, and there is no indication that the video was watched. In other words, the watched videos on clipious are not synced with my invidious instance (but the other way works - a video watched in invidious shows in the history on clipious). Are we talking about the same thing?

I am on invidious version 2024.04.27-eda7444 @ master and clipious version 1.20.0 (compilation 40583)

**Comment by Malakai-DB** [comment]
Just to report back.
I upgraded my phone, and on this one, Clipious correctly marks the videos as watched on the instance.
So it seems the problem I was encountering, was specific to my previous phone (Pixel 5).

But the feature request on this issue is still valid.
I didn't find a way to mark a video as watched without actually opening it.
